kimchi/src/circuits/polynomials/keccak/witness.rs

```diff
@@ -0,0 +1,120 @@
+//! Foreign field multiplication witness computation
```

Comment:
Update to new framework will be easy once it's merged.

Reply:
done!